README.md

```diff
@@ -22,37 +31,49 @@ maximum likelihood inference, perform hypothesis testing with
 Satterthwaite adjusted degrees of freedom, and extract least square
 means estimates by using `emmeans`.
 
-## Main Features
-
-- Responses are assumed normally distributed.
-- Covariances:
-  - Structures: unstructured, Toeplitz, AR1, compound symmetry, and
-    ante-dependence.
-  - Groups: shared covariance structure for all subjects, or group
-    specific covariance structures.
-  - Variances: homogeneous or heterogeneous across time points.
-- Hypothesis testing:
-  - Least square means: `emmeans` package can be used with model outputs
-    to obtain least square means.
-  - Degrees of freedom adjustment: Satterthwaite-adjusted one- and
-    multi-dimensional contrasts.
-- Model inference:
-  - Supports REML and ML.
-  - Supports weights.
-  - Automatic changing of optimizer in the case of non-convergence.
-  - Manual control of optimization routine.
+**Scope:**
+
+-   Continuous responses with normal (but potentially heteroscedastic)
+    residuals.
+-   Only marginal linear models without individual-level random effects
+    are supported
+
+**Main Features:**
+
+-   Flexible covariance specification:
+    -   [Structures](https://openpharma.github.io/mmrm/main/articles/covariance.html):
+        unstructured, Toeplitz, AR1, compound symmetry, and
+        ante-dependence.
+    -   Groups: shared covariance structure for all subjects, or group
+        specific covariance structures.
```

Comment:
I think it is not group specific covariance structures, but group specific covariance estimates

Reply:
Ah yeah that is true. The structure needs to be the same

Reply:
done